reactos/drivers/storage/storahci/storahci.c

```diff
@@ -0,0 +1,745 @@
+/*
+ * PROJECT:        ReactOS Kernel
+ * LICENSE:        GNU GPLv2 only as published by the Free Software Foundation
+ * PURPOSE:        To Implement AHCI Miniport driver targeting storport NT 5.2
+ * PROGRAMMERS:    Aman Priyadarshi (aman.eureka@gmail.com)
+ */
+
+#include "storahci.h"
+
+/**
+ * @name AhciPortInitialize
+ * @implemented
+ *
+ * Initialize port by setting up PxCLB & PxFB Registers
+ *
+ * @param portExtension
+ *
+ * @return
+ * Return true if intialization was successful
+ */
+BOOLEAN AhciPortInitialize(

```

Comment:
In general, our coding style for function is:

RETURNTYPE
FunctionName (
    _In_Sal2_Annotations PTYPE Var,
    _In_Sal2_Annotations PTYPE Var
    )
{
   ...
}

OR:

RETURNTYPE
FunctionName (_In_Sal2_Annotations PTYPE Var,
                         _In_Sal2_Annotations PTYPE Var)
{
...
}

Can you pick one and be consistent? Thanks!